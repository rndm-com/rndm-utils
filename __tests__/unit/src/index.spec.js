const { describe, test } = global;

describe('src/index', () => {
  test('exported', () => {
    const exported = require('../../../src/index');
    expect(Object.keys(exported)).toMatchSnapshot();
  });

  test('exported', () => {
    const exported = require('../../../src');
    expect(Object.keys(exported)).toMatchSnapshot();
  });
});
