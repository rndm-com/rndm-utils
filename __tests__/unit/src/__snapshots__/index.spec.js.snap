// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/index exported 1`] = `
Array [
  "mkDir",
  "readDir",
  "decircular",
  "buildFilesForFolder",
]
`;

exports[`src/index exported 2`] = `
Array [
  "mkDir",
  "readDir",
  "decircular",
  "buildFilesForFolder",
]
`;
